generation_config.yaml

```diff
@@ -1,5 +1,5 @@
 gapic_generator_version: 2.54.0
```

Comment:
@JoeWang1127 @diegomarquezp Which automation is responsible to update this to 2.54.1?

Reply:
That would be https://github.com/googleapis/java-bigtable/blob/main/.github/workflows/update_generation_config.yaml, which is implemented in https://github.com/googleapis/java-bigtable/blob/main/.github/scripts/update_generation_config.sh

Note: this file is templated and should be automatically updated by hermetic build, but we are yet to find out why they (and possibly other files) are not updated in https://github.com/googleapis/sdk-platform-java/issues/3701

Reply:
Note: we could use a tracer to understand why the generator version was not updated by looking at the logs https://github.com/googleapis/java-bigtable/actions/runs/13863998657/job/38798879918

Reply:
The implementation script (update_generation_config.sh) is stale. It should look like https://github.com/googleapis/java-pubsub/blob/main/.github/scripts/update_generation_config.sh

Reply:
I raised https://github.com/googleapis/java-bigtable/pull/2524 as a fix